app/mailers/reminder_mailer.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class ReminderMailer < ApplicationMailer
+  def grant_reviews_reminder(grant:, reviewer:, incomplete_reviews: )
+    @grant              = grant
+    @reviewer           = reviewer
+    @incomplete_reviews = incomplete_reviews
+
+    mail(to: reviewer.email, subject: I18n.t('mailers.reminder.grant_reviews.subject', grant_name: grant.name))
+
+    @incomplete_reviews.each do |review|
+      review.update_column(:reminded_at, Time.now)
```

Comment:
the DB updates should move out of mailer because calling `mail()` doesn't necessarily at the same time deliver the mail. The reminded_at should be updated when `mail.deliver_now()` is called